### PR TITLE
SearchingBox was added to Map Component,  not generated correctly

### PR DESCRIPTION
**Pull request by Szpadzik**

### CODE REVIEW

File: src/components/Detail.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ import React, {Component} from "react";
 
 export default class Details extends Component {
 
+    constructor(props) {
+        super(props);
+    }
+
```
Comment (michalczukm) [inline src/components/Detail.js:12]:
```jsx
navigateToBlock = (block) => {
    this.props.history.push(`../${block.name}`);
}
```

File: src/components/Detail.js [hunk 2]
```diff
@@ -15,6 +19,8 @@ export default class Details extends Component {
                 <div className='details-market' key={marketDetails.id}>
                     <div className="name-bar">
                         <h1 className="name-market">{marketDetails.name}</h1>
+                        <button className="BackButton"
```
Comment (michalczukm) [inline src/components/Detail.js:30]: `<button className="BackButton" onClick={this.navigateToBlock.bind(this, block)}>X</button>`

File: src/components/SearchingMap.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { withLeaflet, MapControl } from "react-leaflet";
+import { OpenStreetMapProvider, GeoSearchControl } from "leaflet-geosearch";
+
+class GeoSearch extends MapControl {
+    constructor(props, context) {
+        super(props);
+    }
+
+    createLeafletElement(opts) {
+        const provider = new OpenStreetMapProvider();
```
Comment (michalczukm) [inline src/components/SearchingMap.js:11]: From what I see we can use `customProvider`
Screenshot 2020-04-08 at 18 05 09

**Review by michalczukm — CHANGES_REQUESTED** [review]
Please fix this stuff about navigation

**Comment by michalczukm** [comment]
Waiting fo changes, @Szpadzik please ping me when you push them ;)

**Comment by Szpadzik** [comment]
@michalczukm, i've corrected the method for back button, now i want to merge my all branches (firebase and search).

**Review by michalczukm — APPROVED** [review]

**Pull request merged**